alb-lambda-rest-api-sam-py/example-pattern.json

```diff
@@ -0,0 +1,72 @@
+{
+  "title": "AWS Lambda REST API and Amazon ALB with Listener Rules",
+  "description": "This configuration employs Python 3.9 and the Serverless Application Model (SAM) CLI to establish an Application Load Balancer with route-based listener rules, paired with an AWS Lambda RESTful API function as the target.",
+  "language": "Python",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "This setup orchestrates the deployment of an Application Load Balancer, configures path-based routes directing traffic to a Python-based AWS Lambda function, and leverages the [AWS Lambda Powertools for Python](https://docs.powertools.aws.dev/lambda/python/latest/core/event_handler/api_gateway/) library. The Lambda function, serving as the target, records details of the incoming ALB event, along with the API and context objects, logging them to an Amazon CloudWatch Logs log group and Amazon X-Ray."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/alb-lambda-rest-api-sam-py",
+      "templateURL": "serverless-patterns/alb-lambda-rest-api-sam-py",
+      "projectFolder": "alb-lambda-rest-api-sam-py",
+      "templateFile": "template.yaml"
+    }
+  },
+  "resources": {
```

Comment:
please include a link to Lambda Powertools (even though there is already a link in the introBox)